scripts/alot.coffee

```diff
@@ -25,5 +25,5 @@ alot = [
   "https://s3.amazonaws.com/hudl-internal-assets/alot/i__m_sad__alot_by_celestialbeast-d3f0uv1.jpg"]
 
 module.exports = (robot) ->
-  robot.hear /alot/i, (msg) ->
+  robot.respond /alot/i, (msg) ->
```

Comment:
hahaha well, the consensus seems to be that hear is annoying.  I also like it, personally.